packages/flutter/test/material/text_field_test.dart

```diff
@@ -7249,11 +7250,11 @@ void main() {
 
     // Tap the TextField to put the cursor into it and bring it into view.
     expect(scrollController.offset, 0.0);
-    await tester.tap(find.byType(TextField));
+    await tester.tapAt(tester.getTopLeft(find.byType(TextField)));
```

Comment:
This seems weird... I don't know why it isn't receiving a tap with the previous line now.  If anyone has a hunch please let me know.